V5 Release note - Productivity part (#879)
akeneo/pim-helpcenter: content/updates/productivity/2-Easier-collaboration-with-increased-import-and-export-capabilities.md

## Export your products with their assets
::: meta-data type="New" ee-only="true" link-to-doc="../articles/exports.html#update-an-export-profile"

It's now possible to export products and their associated assets from the product grid or your product export profiles. Collaboration around product data is more efficient and it will save you time whenever you need to share product information with your internal and external teams.

To do so, you can use our new `With media` option when doing a quick export or using your favorite product export profile. Note, that by default, this option is not enabled.
![Quick export products with their assets](../img/quick-export-products-with-assets.png)

## Import assets with Excel or CSV files
::: meta-data type="New" ee-only="true" link-to-doc="../articles/import-assets.html"

To save you a tremendous amount of time, it is now possible to mass import assets via Excel or CSV files containing all your asset values. Indeed, in our `Import` menu, you can now discover a brand new import profile dedicated to this task. 

![Asset import profile](../img/asset-import-profile.png)

> ::: info

Reviewer: I'd remove the info and maybe add it in the text above. Also, it could be confusing, how do we define "media file": what it is, and what it is not? Not sure what it covers. Or should we just say you can import assets used as main media for your products..

Author: Fixed with Laure

Note that this feature is only available for assets with media file as main media.
:::

## Display labels instead of codes in your product exports
::: meta-data type="New" ee-only="false" link-to-doc="../articles/exports.html#update-an-export-profile"
:::

> In the quick export and your product Excel and CSV export profiles, you can now enable a new `With labels` option. It will allow you to export product data labels (e.g. attribute labels and values, etc.) instead of their code. With an increased readability, it is pretty useful if you plan to share this file to colleagues or external contributors.

Reviewer: suggesting a new sentence because for the quick exports, we don't really "enable" a new option... people choose to export with labels or with codes.
```suggestion
In the quick export and in your product Excel and CSV export profiles, you can now choose whether you want to export your products with their labels or with their codes. The `With labels` option allows you to export product information labels (e.g. attribute labels and option values, etc.) instead of their codes (attribute codes, options codes, etc.). With increased readability, it is pretty useful if you plan to share this file with colleagues or external contributors.
```

![With labels option in the product export profile](../img/labels-in-product-export-profile.png)

Note that you can even choose the locale of the labels.

> ::: info

Reviewer: ```suggestion
```
We can maybe remove this info part? It only applies to product export from the export builder, not quick exports, and with the new sentence it's clear they can choose to export labels or codes.

> If you prefer to stick to the previous format of the export files, no problem, simply leave the button disabled.

Reviewer: ```suggestion
```

> :::

Reviewer: ```suggestion
```
